delta/plugins/storage/src/main/scala/ch/epfl/bluebrain/nexus/delta/plugins/storage/files/Files.scala

```diff
@@ -692,11 +686,11 @@ object Files {
     * Entity definition for [[Files]]
     */
   def definition(implicit
-      clock: Clock[UIO]
+      clock: Clock[IO]
   ): ScopedEntityDefinition[Iri, FileState, FileCommand, FileEvent, FileRejection] =
     ScopedEntityDefinition(
       entityType,
-      StateMachine(None, evaluate, next),
+      StateMachine(None, evaluate(_, _).toBIO[FileRejection], next),
```

Comment:
This is where the automatic widening got me - meant that the route error handling wasn't triggered